test/integration/cert_storage_failed_test.go

```diff
@@ -74,23 +74,27 @@ func getPrecertByName(db *sql.DB, reversedName string) (*x509.Certificate, error
 func TestIssuanceCertStorageFailed(t *testing.T) {
 	os.Setenv("DIRECTORY", "http://boulder.service.consul:4001/directory")
 
-	ctx := context.Background()
-
 	db, err := sql.Open("mysql", vars.DBConnSAIntegrationFullPerms)
 	test.AssertNotError(t, err, "failed to open db connection")
 
-	_, err = db.ExecContext(ctx, `DROP TRIGGER IF EXISTS fail_ready`)
-	test.AssertNotError(t, err, "failed to drop trigger")
-
-	// Make a specific insert into certificates fail, for this test but not others.
-	// To limit the effect to this one test, we make the trigger aware of a specific
-	// hostname used in this test. Since the INSERT to the certificates table
-	// doesn't include the hostname, we look it up in the issuedNames table, keyed
-	// off of the serial.
-	// NOTE: CREATE and DROP TRIGGER do not work in prepared statements. Go's
-	// database/sql will automatically try to use a prepared statement if you pass
-	// any arguments to Exec besides the query itself, so don't do that.
-	_, err = db.ExecContext(ctx, `
+	if os.Getenv("USE_VITESS") == "false" {
+		// This block is only necessary for ProxySQL + MariaDB and can be
+		// deleted once we're fully migrated to Vitess + MySQL 8, where the
+		// trigger is installed via test/vtcomboserver/install_trigger.sh.
```

Comment:
This comment is very likely to be blindly deleted when this whole conditional section is dropped. We should make sure that there's a very prominent comment linking from this test case to install_trigger.sh that will be preserved.